torchx/schedulers/__init__.py

```diff
@@ -19,6 +19,7 @@
     "kubernetes": "torchx.schedulers.kubernetes_scheduler",
     "kubernetes_mcad": "torchx.schedulers.kubernetes_mcad_scheduler",
     "aws_batch": "torchx.schedulers.aws_batch_scheduler",
+    "aws_sagemaker": "torchx.schedulers.aws_sagemaker_scheduler",
```

Comment:
lets add aws_sagemaker in the docs as well:

1. Add `schedulers/aws_sagemaker` in [`docs/source/index.rst`](https://github.com/pytorch/torchx/blob/main/docs/source/index.rst?plain=1#L79)
2. Add a file `aws_sagemaker.rst` in [`docs/source/schedulers`](https://github.com/pytorch/torchx/tree/main/docs/source/schedulers). Take a look at the other scheduler's rst for reference. In general we use sphinx auto-module so that we keep most of the documentation context in code (as doc strings) and use the `rst` to sketch out the layout (e.g. defining and ordering sections).